src/shared/api/types.ts

```diff
@@ -65,7 +65,7 @@ export type ApiUrl = {
   openExternal: (url: string) => Promise<void>
 }
 
-export type Network = 'testnet' | 'chaosnet' | 'mainnet'
+export type Network = 'testnet' | 'mainnet'
```

Comment:
Note: Same type as we have with `Network` in `xchain-client`. However, I would still stick with a custom type here to be independent of `xchain-*` - just in case `xchain-*` will change it for whatever reason in the future.